test/test_sparse_csr.py

```diff
@@ -1256,6 +1259,32 @@ def test_direct_coo_csr_conversion(self, device, dtype):
 
             self.assertEqual(coo_sparse.to_sparse_csr().to_sparse_coo(), coo_sparse)
 
+    @ops(sparse_csr_linalg_solve)
+    def test_linalg_solve_sparse_csr_cusolver(self, device, dtype, op):
+        if device == 'meta':
+            self.skipTest("Skipped!")
+
+        samples = op.sample_inputs(device, dtype)
+
+        for sample in samples:
+            if sample.input.ndim != 2 or sample.args[0].ndim != 1 or sample.args[0].size() == torch.Size([0]):
+                continue
+
+            out = torch.zeros(sample.args[0].size(), dtype=dtype, device=device)
+            if device != 'cpu':
+                with self.assertRaisesRegex(NotImplementedError, "doesn't exist for this backend"):
+                    op(sample.input.to_sparse_csr(), *sample.args, **sample.kwargs, out=out)
+                break
```

Comment:
I am thinking - this can be an expected failure in the OpInfo itself. TODO.

Reply:
My bad, OpInfo doesn't test for Sparse CSR (except unary functions for now). So we'll have to have this check in `test/test_sparse_csr.py`

Reply:
It's fine to not use OpInfo for this test.